### PR TITLE
realease: :rocket: Docusaurus Static Website for EUID

### PR DESCRIPTION
**Pull request by bbuilds**
🛑 🛑  **Please don't merge until approved by @lionell-pack-ttd** 🛑 🛑

### CODE REVIEW

**Comment by bbuilds** [comment]
**Please don't merge until approved by @lionell-pack-ttd**

**Review by lionell-pack-ttd — DISMISSED** [review]
From a tech point of view, this looks good. Approved!

File: README.md
Comment (lionell-pack-ttd) [inline README.md:1]: Can you leave this with the original content for now? This repo will still be the source of documentation until we get all this content moved into Docusaurus.

**Comment by bbuilds** [comment]
sounds good @lionell-pack-ttd  and @genwhittTTD  the current repo files have been merged into this PR

**Review by lionell-pack-ttd — APPROVED** [review]

**Pull request merged**
